src/api/__test__/pagopaApi.integration.test.ts

```diff
@@ -0,0 +1,60 @@
+import { Millisecond } from 'italia-ts-commons/lib/units';
+import { PaymentManagerClient } from '../pagopa';
+import { defaultRetryingFetch } from '../../utils/fetch';
+import * as myFetch from '../../utils/fetch';
+
+// Client for the PagoPA PaymentManager
+describe('Weak Test Suite', () => {
```

Comment:
I suppose we can come up with a better name here :)

Reply:
:-) Yes of course, I'll fix it, but the tests are indeed weak since I've tested only the `getSession`. I'm planning to add tests while we implement the subsequent use cases